dev-itpro/developer/devenv-partial-records.md

```diff
@@ -159,10 +159,8 @@ Like with reports, other fields aren't selected for load, even if they may be us
 
 Partial records are automatically applied based on the page’s metadata for List and ListPart page types that are opened in the web client. As with OData pages, the page's definition is used to select which fields to load. More fields may be required, for example, to enable subpage linking. 
 
-To extend which fields will be loaded, you can use the same approach as with OData pages.
-
 > [!NOTE]
-> OnFindRecord and OnNextRecord triggers conflict with partial record feature, so if either of these triggers is defined in the metadata, the partial record feature won't be applied and it fall back to standard loading behavior.
+> OnFindRecord and OnNextRecord triggers conflict with partial record feature with List and ListPart pages, so if these triggers are defined in the metadata, the partial record feature won't be applied and it fall back to standard loading behavior.
```

Comment:
... falls back to ...

Reply:
Sorry, NKarolak  I re-requested review by mistake. Thanks for your input!